src/testing/testing.md

```diff
@@ -0,0 +1,549 @@
+# Testing Strategy
+
+## 1. What Types of Tests Should We Write?
+
+To fully cover our application and ensure code quality, we need to implement various types of tests:
+
+### *Unit Tests:*
+- **Purpose:** Test individual functions or components in isolation without dependencies on external systems such as APIs or databases.
+- **Usage:** For small units like functions, classes, or components.
+- **Examples:**
+    - Utility functions that perform complex calculations.
+    - Testing all composables.
+    - Verifying that a form validates as expected when the user inputs data.
+
+```typescript
+// Example of a Unit Test for a Login Function
+test('users can successfully log in', async ({ page }) => {
```

Comment:
dit is wel geen goede voorbeeld van een unit test. Hier kunt ge beter een voorbeeld nemen van vitest die een bepaalde functie gaat testen